scripts/product_migration.py

```diff
@@ -18,9 +18,6 @@
 PRODUCT_NEW = "MeerKATVisibilityProduct"
 FLAGS_NEW = "MeerKATFlagProduct"
 
-STARTDATE = os.environ.get("STARTDATE", "2010-01-01T00:00:00Z")
```

Comment:
If you could trace back e.g. an imaging product to its vis data product, you could use the start and end time of that product instead. Or are there products that don't ultimately derive from CBF data?